e2e/pages/app/column-picker/column-picker.testpage.component.html

```diff
@@ -3,12 +3,14 @@
     [deselectedTitleTemplate]="showCustomTitles ? title : null"
     [(selected)]="selected"
     [(deselected)]="deselected"
+    [(groups)]="groups"
     [locked]="locked">
 </ux-column-picker>
 
 <ng-template #title>Custom Title</ng-template>
 
 <p id="selected-list">{{ selected | json }}</p>
 <p id="deselected-list">{{ deselected | json }}</p>
+<p id="group-settings-list">{{ groups | json }}</p>
```

Comment:
This will always be the value provided by the test case, so it doesn't provide any coverage.